sparkle/src/components/Tabs.tsx

```diff
@@ -50,6 +50,8 @@ const TabsTrigger = React.forwardRef<
     icon?: React.ComponentType;
     isLoading?: boolean;
     buttonVariant?: React.ComponentProps<typeof Button>["variant"];
+    isCounter?: boolean;
+    counterValue?: string;
```

Comment:
Could we also do get the types through `React.ComponentProps<typeof Button>....`? For the sake of consistency